src/java.base/share/classes/java/lang/NumberFormatException.java

```diff
@@ -82,4 +92,8 @@ static NumberFormatException forCharSequence(CharSequence s,
                 + (errorIndex - beginIndex) + " in: \""
                 + s.subSequence(beginIndex, endIndex) + "\"");
     }
+
+    static NumberFormatException emptyInput() {
```

Comment:
A better method name might be nullInput, since an empty (but non null) string doesn’t use this